### ISSUE TITLE
Game crash after trying to load INMIS on 1.16 release

### ISSUE THREAD

**Issue by NxD136** [Draylar/inmis#21]
The game keeps on crashing with this error message when trying to load into Minecraft : 

```
The game crashed whilst initializing game
Error: java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'inmis'!
```

The only two mods I have installed are the latest INMIS mod (1.0.0-1.16) and fabric API (0.13.1+build.370-1.16).

Attached is the crash report.
[crash-2020-06-23_18.19.34-client.txt](https://github.com/Draylar/inmis/files/4822274/crash-2020-06-23_18.19.34-client.txt)

**Comment by ArchieBeepBoop** [comment]
Can confirm.
https://paste.ee/p/7QWzL

**Comment by NxD136** [comment]
Thanks for the confirmation.

Any ETA on a fix? No pressure really, just curious is all :)

Thanks for the amazing mod and your swift replies!

**Comment by ArchieBeepBoop** [comment]
I'm not the creator lol.

**Comment by Draylar** [comment]
looking into it in one sec, thanks for the report

**Comment by Draylar** [comment]
Okay, I've fixed this, but I still need to fix #22. I'm just pulling the 1.0.0 release offline until I figure this out.

**Comment by Draylar** [comment]
Fixed in 1.0.2 ([CF](https://www.curseforge.com/minecraft/mc-mods/inmis/files/2988043), [GH](https://github.com/Draylar/inmis/releases/tag/1.0.2-1.16)).
